src/actions/helpers/getPlayersHelper.js

```diff
@@ -0,0 +1,22 @@
+import socket from '../../socketHandlers/index';
+
+const SOCKET_EVENT_TYPES = {
+  EMIT: 'getPlayers',
+  ON: 'resultGetPlayers'
+}
+
+const getPlayersPromise = (roomId) => {
+  return new Promise((resolve, reject) => {
+    socket.on(SOCKET_EVENT_TYPES.ON, (data) => {
+      if(data.result.error) {
+        resolve(data.result.error);
+      } else {
+        const playerIds = data.result.data;
+        reject(playerIds);
```

Comment:
Don't call reject here because this is success pattern.

`reject` should be called when that is error. On the other hand, resolve is used when that is success.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise